src/vendor/convert.js

```diff
@@ -107,7 +107,10 @@ const toURL = { '+': '-', '/': '_', '=': '' };
 convert from bytes to base64 (encoded to be URL safe)
  */
 function bytesToBase64URL(bytes) {
-  return window.btoa(String.fromCharCode(...Array.from(new Uint8Array(bytes))))
+  if (isString(bytes)) {
+    return null.f();
```

Comment:
This is not clear.  You can

    raise new TypeError("with some explanation")

or you can use the `Log` library

    raise Log.error("expecting array of ints, not {{type}}", {type: typeof bytes})

Reply:
It need one more dependency into the package to throw new TypeError (@babel/plugin-proposal-throw-expressions).
For logging the error it will need to disable no-console rule for the line of code.

Which option would be better?

Reply:
Sorry for the delay.  I see I used Python's `raise` instead of ES6 `throw`.

Use 

    Log.error("expecting array of ints, not {{type}}", {type: typeof bytes})

which will not need the dependency, it will log the problem, and throw and error.

Reply:
Hi @wicked-kiD , I'm unsure why you marked this as resolved. I could be mistaken, but it doesn't appear you've made the changes that Kyle requested

Reply:
> Hi @wicked-kiD , I'm unsure why you marked this as resolved. I could be mistaken, but it doesn't appear you've made the changes that Kyle requested

Hi @kimberlythegeek ! I did this changes, you can see them at 111-113 lines at convert.js. They are looking a bit different because example was not on JS. Also I added test case for this at lines 53, 62-64 at convert.test.js